src/middlewared/middlewared/plugins/jail.py

```diff
@@ -242,7 +252,7 @@ def check_jail_existence(self, jail, skip=True):
         try:
             iocage = ioc.IOCage(skip_jails=skip, jail=jail)
             jail, path = iocage.__check_jail_existence__()
-        except SystemExit:
+        except CallError:
```

Comment:
This needs to be both SystemExit (for middlewared console being up) and RuntimeError (the rest of the time)